### ISSUE TITLE
Describe Stored Query test failure for 2.0.2

### ISSUE THREAD

**Issue by du002cl** [opengeospatial/ets-wfs20#62]
We are seeing a failure when running against version 2.0.2

> **Name**: describeStoredQuery_GetFeatureById
> **Description**: See ISO 19142: 7.9.3.6
> **Signature**: describeStoredQuery_GetFeatureById(org.opengis.cite.iso19142.ProtocolBinding)[pri:0, instance:org.opengis.cite.iso19142.simple.DescribeStoredQueriesTests@720a9d9f]
> **Parameters**: GET

**Attributes**:
request
`service=WFS&version=2.0.0&request=DescribeStoredQueries&storedquery_id=http://www.opengis.net/def/query/OGC-WFS/0/GetFeatureById`
response
```
<ows:ExceptionReport xmlns:ows="http://www.opengis.net/ows/1.1" version="2.0.0">
	<ows:Exception exceptionCode="InvalidParameterValue" locator="DescribeStoredQueries">
		<ows:ExceptionText>Invalid Stored Query Id value: http://www.opengis.net/def/query/OGC-WFS/0/GetFeatureById</ows:ExceptionText>
	</ows:Exception>
</ows:ExceptionReport>
```

As you can see the test is using version 2.0.0, even thought the initial GetCapabilities is 2.0.2. The test fails because it is using the new 2.0.2 URI format which was not in place for 2.0.0. We could change our server code to handle this case but does not seem that should be necessary. 

This test passes on 2.0.0

All the above is also true for the POST version of this test.

Here is the session: http://cite.opengeospatial.org/teamengine/reports/mideveloper/s0213/html/

**Comment by du002cl** [comment]
**Invoke Get Feature By Id** and **Invoke Get Feature By Id With Unknown I D** are also failing for the same reason as the test listed above. All information can be seen in the session linked in the issue.

**Comment by lgoltz** [comment]
@du002cl Can you provide a service for testing?

**Comment by lgoltz** [comment]
As far as I understand the parameter `storedquery_id` is correct for 2.0.2 (see 7.9.3.6 GetFeatureById stored query in http://docs.opengeospatial.org/is/09-025r2/09-025r2.html#24). But the version requested is 2.0.0, this should be 2.0.2 if sut is WFS 2.0.2. Fixed in #88
